### ISSUE TITLE
Add sparkline examples

### ISSUE THREAD

**Issue by heruan** [vaadin/vaadin-charts-flow#193]
In the Vaadin website there is this sparkline page: https://vaadin.com/components/vaadin-sparkline.

But I can't find code example to obtain that, since the links point to a page with many examples but no sparkline ones.

**Comment by johannesh2** [comment]
Thank you for the report. That page should be removed as vaadin-sparkline is not available anymore in 6.0. Website is currently lagging behind the releases and is describing vaadin-charts 5 features. Sparkline was basically not a separate chart-type, but a pre-configured line chart. Makes sense to implement it to charts 6.x too. I will create a ticket to the website team for removing the page. I created #194 for adding it to Charts 6.

**Comment by johannesh2** [comment]
@oluwasayo you had implemented this in the Speedment demo, right? Would it make sense to add that code to documentation?

**Comment by oluwasayo** [comment]
Yes we implemented a [sparkline in Speedment demo](https://github.com/vaadin/stocks-demo/blob/master/application/src/main/webapp/frontend/src/sparkline-chart.html). Perhaps we can create a doc chapter to discuss it in detail.
